src/fare-contracts/FareContractInfo.tsx

```diff
@@ -101,6 +109,10 @@ export const FareContractInfo = ({
     userProfiles,
   );
 
+  const [startPlaceName, endPlaceName] = [startPointRef, endPointRef].map(
+    (pointRef) => boatStopPoints.find((bSP) => bSP.id == pointRef)?.name || '',
```

Comment:
Yes, meant to be short for boatStopPoints. However you might prefer a different style? E.g. bsp?

Reply:
I think the most important thing is to try keeping it the same way as in the rest of the code. Tried to look up some examples, and found that FareContract typically is cased `fc`, so I think bsp would work great! 👌 

Using the whole word is of course also an option since the abbreviation is only used twice.

Reply:
Agreed, changing to bsp :+1: